web/src/components/Section.js

```diff
@@ -32,7 +45,7 @@ Section.defaultProps = {
   resource: null
 };
 
-const Chunk = ({ children, resource }) => {
+const SubsectionChunk = ({ children, resource }) => {
```

Comment:
This is definitely a better name.  Still not completely intuitive, but much better.  👍